docs/targets/solana.rst

```diff
@@ -74,6 +74,28 @@ are `some examples <https://solana-labs.github.io/solana-solidity.js/>`_. There
 `solang's integration tests <https://github.com/hyperledger-labs/solang/tree/main/integration/solana>`_.
 
 
+.. _value_transfer:
+
+Transfering native value with a function call
+_____________________________________________
+
+The Solidity langauge on Ethereum allows value transfers with an external call. Solana
+Cross Program Invocation (CPI) does not support this. Hypothetically, the caller
+could transfer native balance and then inform the callee about the amount transferred
+by specifying this in the instruction data. However, it would be trivial to forge
+this. So, this is not supported at all. This means that:
+
+ - Specifying `value:`` on an external call is not permitted
+ - The `payable` keyword has no effect
+ - `msg.value` is not supported
+
+Receive function
+________________
+
+In Solidity, a `receive()` function can be defined which gets called, whenever the
```

Comment:
`function can be defined which gets called`
This does not sound right.

Reply:
Rewritten